src/core/AnimatedStopClock.js

```diff
@@ -9,7 +9,9 @@ export default class AnimatedStopClock extends AnimatedNode {
     super({ type: 'clockStop', clock: clockNode.__nodeID });
     invariant(
       clockNode instanceof AnimatedClock,
-      'Node is not of an AnimatedClock type'
+      'Reanimated: Animated.stopClock argument should be of type AnimatedClock but got %s, NodeID: %s',
+      typeof clockNode,
```

Comment:
and `typeof clockNode` will probably be `object` so it won't help a lot. Also, `typeof clockNode` is on `object` as well. Is suggest simplifying to `Reanimated: Animated.stopClock argument should be of type AnimatedClock`.